packages/core-components/src/components/CodeSnippet/CodeSnippet.tsx

```diff
@@ -13,16 +13,17 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-import React from 'react';
-import { useTheme } from '@material-ui/core/styles';
 import { BackstageTheme } from '@backstage/theme';
-import { CopyTextButton } from '../CopyTextButton';
-import type {} from 'react-syntax-highlighter';
-import { default as LightAsync } from 'react-syntax-highlighter/dist/esm/light-async';
+import Box from '@material-ui/core/Box';
+import { useTheme } from '@material-ui/core/styles';
+import React from 'react';
+import LightAsync from 'react-syntax-highlighter/dist/esm/light-async';
 import dark from 'react-syntax-highlighter/dist/esm/styles/hljs/dark';
 import docco from 'react-syntax-highlighter/dist/esm/styles/hljs/docco';
 
+import { CopyTextButton } from '../CopyTextButton';
+
+import type {} from 'react-syntax-highlighter';
```

Comment:
see https://github.com/backstage/backstage/pull/13808 - i think you should revert this reordering of imports

Reply:
Done, will have to check what my import sorter does from now on.